### ISSUE TITLE
error  undefined method `key?' for "Not Found":String

### ISSUE THREAD

**Issue by yahonda** [mtsmfm/docker-rails-dev-box#1]
I've been looking for something like rails-dev-box on Docker, not using Vagrant and found this repository. Thanks for the information at RubyKaigi in Hiroshima.

I'm getting `error  undefined method `key?' for "Not Found":String` as follows.

```ruby
$ pwd
/Users/yahonda/git
$ git clone https://github.com/rails/rails
Cloning into 'rails'...
remote: Counting objects: 607303, done.
remote: Compressing objects: 100% (112/112), done.
remote: Total 607303 (delta 54), reused 48 (delta 26), pack-reused 607165
Receiving objects: 100% (607303/607303), 151.68 MiB | 1.98 MiB/s, done.
Resolving deltas: 100% (447803/447803), done.
$ cd rails
$ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/setup | bash
+ mkdir -p .git/info
+ download docker-compose.yml
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/docker-compose.yml
+ download docker-sync.yml
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/docker-sync.yml
+ download Dockerfile
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/Dockerfile
+ write_line .git/info/exclude /docker-compose.yml
+ touch .git/info/exclude
+ grep -xq /docker-compose.yml .git/info/exclude
+ echo /docker-compose.yml
+ write_line .git/info/exclude /docker-sync.yml
+ touch .git/info/exclude
+ grep -xq /docker-sync.yml .git/info/exclude
+ echo /docker-sync.yml
+ write_line .git/info/exclude /Dockerfile
+ touch .git/info/exclude
+ grep -xq /Dockerfile .git/info/exclude
+ echo /Dockerfile
+ docker-sync start
       error  undefined method `key?' for "Not Found":String
$
```

Here are required software versions. 
* macOS High Sierra 10.13 Beta (17A360a)
```
$ docker -v
Docker version 17.06.2-ce, build cec0b72
$ docker-sync -v
0.4.6
$ docker-compose -v
docker-compose version 1.14.0, build c7bdf9e
```

**Comment by mtsmfm** [comment]
@yahonda 

Sorry, I missed pushing some files 🙇 
Could you try again?

**Comment by yahonda** [comment]
Thanks for the update.  No need to say sorry. 

I'm getting the same error somehow.

```ruby
$ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/setup | bash
+ -z ''
bash: line 5: -z: command not found
+ mkdir -p .git/info
+ download docker-compose.yml
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw//docker-compose.yml
+ download docker-sync.yml
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw//docker-sync.yml
+ download Dockerfile
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw//Dockerfile
+ download activerecord/test/config.yml
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw//activerecord/test/config.yml
+ write_line .git/info/exclude /docker-compose.yml
+ touch .git/info/exclude
+ grep -xq /docker-compose.yml .git/info/exclude
+ write_line .git/info/exclude /docker-sync.yml
+ touch .git/info/exclude
+ grep -xq /docker-sync.yml .git/info/exclude
+ write_line .git/info/exclude /Dockerfile
+ touch .git/info/exclude
+ grep -xq /Dockerfile .git/info/exclude
+ docker volume create rails-rails-sync
rails-rails-sync
++ docker container create -v rails-rails-sync:/app_sync busybox
Unable to find image 'busybox:latest' locally
latest: Pulling from library/busybox
03b1be98f3f9: Pulling fs layer
03b1be98f3f9: Verifying Checksum
03b1be98f3f9: Download complete
03b1be98f3f9: Pull complete
Digest: sha256:99ccecf3da28a93c063d5dddcdf69aeed44826d0db219aabc3d5178d47649dfa
Status: Downloaded newer image for busybox:latest
+ container=0d07d7418358a45f0d41eb681545a12f909cb8502e76ec5486dc91d4c211a12e
+ docker cp . 0d07d7418358a45f0d41eb681545a12f909cb8502e76ec5486dc91d4c211a12e:/app_sync
+ docker container rm 0d07d7418358a45f0d41eb681545a12f909cb8502e76ec5486dc91d4c211a12e
0d07d7418358a45f0d41eb681545a12f909cb8502e76ec5486dc91d4c211a12e
+ docker-sync start
       error  undefined method `key?' for "Not Found":String
$
```

Recently, macOS has been updated to High Sierra 17A362a.

**Comment by mtsmfm** [comment]
@yahonda I noticed some mistakes so I fixed and introduced CI (#3)
I hope it will work well now!

**Comment by yahonda** [comment]
Thanks for the update. Now it is working fine.

<details>

```ruby
$  curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/setup | bash
+ '[' -z '' ']'
+ BRANCH=master
+ mkdir -p .git/info
+ download docker-compose.yml
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/docker-compose.yml
+ download docker-sync.yml
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/docker-sync.yml
+ download Dockerfile
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/Dockerfile
+ download activerecord/test/config.yml
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/activerecord/test/config.yml
+ write_line .git/info/exclude /docker-compose.yml
+ touch .git/info/exclude
+ grep -xq /docker-compose.yml .git/info/exclude
+ write_line .git/info/exclude /docker-sync.yml
+ touch .git/info/exclude
+ grep -xq /docker-sync.yml .git/info/exclude
+ write_line .git/info/exclude /Dockerfile
+ touch .git/info/exclude
+ grep -xq /Dockerfile .git/info/exclude
+ docker-compose down
Removing network rails_default
WARNING: Network rails_default not found.
+ docker-sync clean
bash: line 32: docker-sync: command not found
+ true
+ docker volume create rails-rails-sync
rails-rails-sync
++ docker container create -v rails-rails-sync:/app_sync busybox
+ container=a83c8f746326c358cc0d6381da996e3feb8b8afdb6fc4725a9444ee49dea9439
+ docker cp . a83c8f746326c358cc0d6381da996e3feb8b8afdb6fc4725a9444ee49dea9439:/app_sync
+ docker container rm a83c8f746326c358cc0d6381da996e3feb8b8afdb6fc4725a9444ee49dea9439
a83c8f746326c358cc0d6381da996e3feb8b8afdb6fc4725a9444ee49dea9439
+ docker run --rm -v rails-rails-sync:/app_sync busybox chown -R 1000 /app_sync
+ docker-sync start
bash: line 40: docker-sync: command not found
myretina:rails yahonda$ which docker-sync
myretina:rails yahonda$ docker-sync
-bash: docker-sync: command not found
myretina:rails yahonda$ ruby -v
ruby 2.3.3p222 (2016-11-21 revision 56859) [universal.x86_64-darwin17]
myretina:rails yahonda$ gem install docker-sync
Fetching: docker-sync-0.4.6.gem (100%)
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /Library/Ruby/Gems/2.3.0 directory.
myretina:rails yahonda$ sudo gem install docker-sync
Password:
Sorry, try again.
Password:
Successfully installed docker-sync-0.4.6
Parsing documentation for docker-sync-0.4.6
Done installing documentation for docker-sync after 0 seconds
1 gem installed
myretina:rails yahonda$  curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/setup | bash
+ '[' -z '' ']'
+ BRANCH=master
+ mkdir -p .git/info
+ download docker-compose.yml
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/docker-compose.yml
+ download docker-sync.yml
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/docker-sync.yml
+ download Dockerfile
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/Dockerfile
+ download activerecord/test/config.yml
+ curl -sL https://github.com/mtsmfm/docker-rails-dev-box/raw/master/activerecord/test/config.yml
+ write_line .git/info/exclude /docker-compose.yml
+ touch .git/info/exclude
+ grep -xq /docker-compose.yml .git/info/exclude
+ write_line .git/info/exclude /docker-sync.yml
+ touch .git/info/exclude
+ grep -xq /docker-sync.yml .git/info/exclude
+ write_line .git/info/exclude /Dockerfile
+ touch .git/info/exclude
+ grep -xq /Dockerfile .git/info/exclude
+ docker-compose down
Removing network rails_default
WARNING: Network rails_default not found.
+ docker-sync clean
Status: Downloaded newer image for eugenmayer/unison:hostsync_0.2
     success  Finished cleanup. Removed stopped, removed sync container and removed their volumes
+ docker volume create rails-rails-sync
rails-rails-sync
++ docker container create -v rails-rails-sync:/app_sync busybox
+ container=87da425df0b3887f9f3e0d6a063b1a57833d30680fbc505abb1422cb1665967a
+ docker cp . 87da425df0b3887f9f3e0d6a063b1a57833d30680fbc505abb1422cb1665967a:/app_sync
+ docker container rm 87da425df0b3887f9f3e0d6a063b1a57833d30680fbc505abb1422cb1665967a
87da425df0b3887f9f3e0d6a063b1a57833d30680fbc505abb1422cb1665967a
+ docker run --rm -v rails-rails-sync:/app_sync busybox chown -R 1000 /app_sync
+ docker-sync start
          ok  Starting native_osx for sync rails-rails-sync
          ok  rails-rails-sync container not running
          ok  creating rails-rails-sync container
          ok  Starting precopy
Contacting server...
Looking for changes
Reconciling changes
Propagating updates
UNISON 2.48.4 started propagating changes at 08:03:12.41 on 24 Sep 2017
[BGN] Copying .docker-sync from /host_sync to /app_sync
[END] Copying .docker-sync
UNISON 2.48.4 finished propagating changes at 08:03:12.41 on 24 Sep 2017
Saving synchronizer state
Synchronization complete at 08:03:12  (1 item transferred, 0 skipped, 0 failed)
real	0m 10.91s
user	0m 0.84s
sys	0m 1.25s
          ok  Starting container
     command  docker run -d -v rails-rails-sync:/app_sync -v /Users/yahonda/git/rails:/host_sync -e HOST_VOLUME=/host_sync -e APP_VOLUME=/app_sync -e TZ=${TZ-`readlink /etc/localtime | sed -e 's,/usr/share/zoneinfo/,,'`} -e UNISON_EXCLUDES="" -e UNISON_ARGS="" -e UNISON_SYNC_PREFER="-prefer /host_sync" -e OWNER_UID="1000"  --name rails-rails-sync eugenmayer/unison:hostsync_0.2
          ok  starting initial sync of rails-rails-sync
     success  Sync container started
     success  Starting Docker-Sync in the background
+ docker-compose build
postgres uses an image, skipping
Building ruby
Step 1/7 : FROM ruby:2.4.1
2.4.1: Pulling from library/ruby
aa18ad1a0d33: Pull complete
15a33158a136: Pull complete
f67323742a64: Pull complete
c4b45e832c38: Pull complete
c1d1736737e7: Pull complete
c1dfd7e8047d: Pull complete
54f441bf1206: Pull complete
b20cc84e5780: Pull complete
Digest: sha256:b6d5b373153e87809d2dd067bfda8a88e901731c3efec0ca640e9a49ffcdd26d
Status: Downloaded newer image for ruby:2.4.1
 ---> e7ca4a0b5b6d
Step 2/7 : RUN apt-get update -q &&   apt-get install -y --no-install-recommends less sqlite3 libsqlite3-dev nodejs
 ---> Running in 8e10a06d9565
Get:1 http://security.debian.org jessie/updates InRelease [63.1 kB]
Get:2 http://security.debian.org jessie/updates/main amd64 Packages [546 kB]
Ign http://deb.debian.org jessie InRelease
Get:3 http://deb.debian.org jessie-updates InRelease [145 kB]
Get:4 http://deb.debian.org jessie Release.gpg [2373 B]
Get:5 http://deb.debian.org jessie Release [148 kB]
Get:6 http://deb.debian.org jessie-updates/main amd64 Packages [23.1 kB]
Get:7 http://deb.debian.org jessie/main amd64 Packages [9063 kB]
Fetched 9991 kB in 58s (170 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
libsqlite3-dev is already the newest version.
The following extra packages will be installed:
  libc-ares2 libv8-3.14.5
Suggested packages:
  sqlite3-doc
The following NEW packages will be installed:
  less libc-ares2 libv8-3.14.5 nodejs sqlite3
0 upgraded, 5 newly installed, 0 to remove and 8 not upgraded.
Need to get 2216 kB of archives.
After this operation, 7897 kB of additional disk space will be used.
Get:1 http://deb.debian.org/debian/ jessie/main libc-ares2 amd64 1.10.0-2+deb8u2 [72.5 kB]
Get:2 http://deb.debian.org/debian/ jessie/main less amd64 458-3 [124 kB]
Get:3 http://deb.debian.org/debian/ jessie/main libv8-3.14.5 amd64 3.14.5.8-8.1 [1269 kB]
Get:4 http://deb.debian.org/debian/ jessie/main sqlite3 amd64 3.8.7.1-1+deb8u2 [101 kB]
Get:5 http://deb.debian.org/debian/ jessie/main nodejs amd64 0.10.29~dfsg-2 [648 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 2216 kB in 13s (164 kB/s)
Selecting previously unselected package libc-ares2:amd64.
(Reading database ... 21171 files and directories currently installed.)
Preparing to unpack .../libc-ares2_1.10.0-2+deb8u2_amd64.deb ...
Unpacking libc-ares2:amd64 (1.10.0-2+deb8u2) ...
Selecting previously unselected package less.
Preparing to unpack .../archives/less_458-3_amd64.deb ...
Unpacking less (458-3) ...
Selecting previously unselected package libv8-3.14.5.
Preparing to unpack .../libv8-3.14.5_3.14.5.8-8.1_amd64.deb ...
Unpacking libv8-3.14.5 (3.14.5.8-8.1) ...
Selecting previously unselected package sqlite3.
Preparing to unpack .../sqlite3_3.8.7.1-1+deb8u2_amd64.deb ...
Unpacking sqlite3 (3.8.7.1-1+deb8u2) ...
Selecting previously unselected package nodejs.
Preparing to unpack .../nodejs_0.10.29~dfsg-2_amd64.deb ...
Unpacking nodejs (0.10.29~dfsg-2) ...
Processing triggers for mime-support (3.58) ...
Setting up libc-ares2:amd64 (1.10.0-2+deb8u2) ...
Setting up less (458-3) ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
Setting up libv8-3.14.5 (3.14.5.8-8.1) ...
Setting up sqlite3 (3.8.7.1-1+deb8u2) ...
Setting up nodejs (0.10.29~dfsg-2) ...
update-alternatives: using /usr/bin/nodejs to provide /usr/bin/js (js) in auto mode
Processing triggers for libc-bin (2.19-18+deb8u10) ...
 ---> e63a1ed17bed
Removing intermediate container 8e10a06d9565
Step 3/7 : RUN useradd --create-home --user-group --uid 1000 app &&   mkdir -p /app/vendor/bundle &&   chown -R app /app
 ---> Running in 5cc1cf1e84c9
 ---> 0028fabe0230
Removing intermediate container 5cc1cf1e84c9
Step 4/7 : USER app
 ---> Running in fb6731a68137
 ---> 7daac2a9e8e7
Removing intermediate container fb6731a68137
Step 5/7 : WORKDIR /app
 ---> 4b093876d98b
Removing intermediate container e0138db6dcc4
Step 6/7 : ENV LANG C.UTF-8 LC_ALL C.UTF-8 BUNDLE_JOBS 4 BUNDLE_PATH /app/vendor/bundle
 ---> Running in 08b405072845
 ---> bf45f1ec2c0a
Removing intermediate container 08b405072845
Step 7/7 : CMD sh -c trap : TERM INT; sleep infinity & wait
 ---> Running in 8e055a7d097c
 ---> 5d4cd06ac213
Removing intermediate container 8e055a7d097c
Successfully built 5d4cd06ac213
Successfully tagged rails_ruby:latest
+ docker-compose up -d
Creating network "rails_default" with the default driver
Creating volume "rails_home" with default driver
Creating volume "rails_bundle" with default driver
Pulling postgres (postgres:9.4-alpine)...
9.4-alpine: Pulling from library/postgres
019300c8a437: Pull complete
885fa9f8b950: Pull complete
df6a6308043b: Pull complete
0bd5b49a0af9: Pull complete
aae46913163b: Pull complete
2cff2a845d54: Pull complete
e5b147178518: Pull complete
6be20fce08c3: Pull complete
6409185e6ecf: Pull complete
Digest: sha256:facd9bf3d02a39c9339fb95a2c60bd2a1f6898180e38ffc6fafd8ad325482db1
Status: Downloaded newer image for postgres:9.4-alpine
Creating rails_postgres_1 ...
Creating rails_postgres_1 ... done
Creating rails_ruby_1 ...
Creating rails_ruby_1 ... done
+ sleep 15
+ docker-compose exec -T ruby bundle install
Fetching gem metadata from https://rubygems.org/...........
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rubygems.org/..
Fetching https://github.com/rails/arel.git
Fetching https://github.com/matthewd/rb-inotify.git
Fetching https://github.com/QueueClassic/queue_classic.git
Fetching https://github.com/rails/sass-rails.git
Fetching https://github.com/robin850/sdoc.git
Fetching https://github.com/matthewd/websocket-client-simple.git
Fetching rake 12.0.0
Fetching concurrent-ruby 1.0.5
Fetching i18n 0.8.6
Installing rake 12.0.0
Fetching minitest 5.10.3
Installing concurrent-ruby 1.0.5
Fetching thread_safe 0.3.6
Installing minitest 5.10.3
Fetching builder 3.2.3
Installing thread_safe 0.3.6
Installing builder 3.2.3
Fetching erubi 1.6.1
Fetching mini_portile2 2.2.0
Installing erubi 1.6.1
Installing i18n 0.8.6
Fetching rack 2.0.3
Fetching nio4r 2.1.0
Installing mini_portile2 2.2.0
Fetching websocket-extensions 0.1.2
Installing nio4r 2.1.0 with native extensions
Installing websocket-extensions 0.1.2
Installing rack 2.0.3
Fetching mime-types-data 3.2016.0521
Using arel 9.0.0.alpha from https://github.com/rails/arel.git (at master@42510bf)
Fetching public_suffix 2.0.5
Fetching amq-protocol 2.2.0
Installing amq-protocol 2.2.0
Fetching ast 2.3.0
Installing public_suffix 2.0.5
Fetching aws-partitions 1.20.0
Installing aws-partitions 1.20.0
Fetching aws-sigv4 1.0.1
Installing ast 2.3.0
Fetching jmespath 1.3.1
Installing mime-types-data 3.2016.0521
Installing aws-sigv4 1.0.1
Fetching multipart-post 2.0.0
Fetching beaneater 1.0.0
Installing multipart-post 2.0.0
Fetching dante 0.2.0
Installing dante 0.2.0
Fetching bcrypt 3.1.11
Installing beaneater 1.0.0
Fetching benchmark-ips 2.7.2
Installing bcrypt 3.1.11 with native extensions
Installing benchmark-ips 2.7.2
Installing jmespath 1.3.1
Fetching blade-qunit_adapter 2.0.1
Fetching coffee-script-source 1.12.2
Fetching execjs 2.7.0
Installing execjs 2.7.0
Fetching curses 1.0.2
Installing coffee-script-source 1.12.2
Fetching eventmachine 1.2.5
Installing blade-qunit_adapter 2.0.1
Fetching cookiejar 0.3.3
Installing curses 1.0.2 with native extensions
Installing cookiejar 0.3.3
Installing eventmachine 1.2.5 with native extensions
Fetching http_parser.rb 0.6.0
Installing http_parser.rb 0.6.0 with native extensions
Fetching multi_json 1.12.1
Installing multi_json 1.12.1
Fetching daemons 1.2.4
Fetching thor 0.20.0
Fetching useragent 0.16.8
Installing thor 0.20.0
Fetching ffi 1.9.18
Installing useragent 0.16.8
Fetching rubyzip 1.2.1
Installing rubyzip 1.2.1
Fetching msgpack 1.1.0
Installing msgpack 1.1.0 with native extensions
Installing daemons 1.2.4
Installing ffi 1.9.18 with native extensions
Using bundler 1.15.4
Fetching byebug 9.0.6
Fetching mini_mime 0.1.4
Fetching method_source 0.8.2
Installing mini_mime 0.1.4
Fetching connection_pool 2.2.1
Installing method_source 0.8.2
Installing byebug 9.0.6 with native extensions
Fetching dalli 2.7.6
Installing connection_pool 2.2.1
Installing dalli 2.7.6
Fetching declarative 0.0.9
Fetching declarative-option 0.1.0
Fetching digest-crc 0.4.1
Installing declarative-option 0.1.0
Fetching hiredis 0.6.1
Installing declarative 0.0.9
Fetching erubis 2.7.0
Installing hiredis 0.6.1 with native extensions
Installing digest-crc 0.4.1
Installing erubis 2.7.0
Fetching event_emitter 0.2.6
Fetching jwt 1.5.6
Fetching little-plugger 1.1.4
Installing jwt 1.5.6
Fetching memoist 0.16.0
Installing event_emitter 0.2.6
Fetching os 0.9.6
Installing little-plugger 1.1.4
Fetching httpclient 2.8.3
Installing os 0.9.6
Installing memoist 0.16.0
Fetching uber 0.1.0
Fetching retriable 3.1.1
Installing uber 0.1.0
Installing retriable 3.1.1
Fetching json 2.1.0
Fetching mustache 1.0.5
Installing mustache 1.0.5
Fetching libxml-ruby 3.0.0
Installing json 2.1.0 with native extensions
Installing libxml-ruby 3.0.0 with native extensions
Installing httpclient 2.8.3
Fetching rb-fsevent 0.10.2
Installing rb-fsevent 0.10.2
Fetching ruby_dep 1.5.0
Fetching metaclass 0.0.4
Fetching mini_magick 4.8.0
Installing mini_magick 4.8.0
Fetching path_expander 1.0.2
Installing metaclass 0.0.4
Fetching mono_logger 1.1.0
Installing path_expander 1.0.2
Fetching mustermann 1.0.0
Installing ruby_dep 1.5.0
Fetching mysql2 0.4.9
Installing mono_logger 1.1.0
Fetching parallel 1.12.0
Installing mysql2 0.4.9 with native extensions
Installing parallel 1.12.0
Installing mustermann 1.0.0
Fetching powerpack 0.1.1
Fetching pg 0.19.0
Fetching puma 3.9.1
Installing powerpack 0.1.1
Fetching que 0.14.0
Installing puma 3.9.1 with native extensions
Installing pg 0.19.0 with native extensions
Installing que 0.14.0
Fetching racc 1.4.14
Fetching rdoc 5.1.0
Fetching redcarpet 3.2.3
Installing racc 1.4.14 with native extensions
Fetching redis 3.3.3
Installing redis 3.3.3
Fetching tilt 2.0.8
Installing redcarpet 3.2.3 with native extensions
Installing tilt 2.0.8
Installing rdoc 5.1.0
Fetching ruby-progressbar 1.8.1
Fetching unicode-display_width 1.3.0
Fetching sequel 4.49.0
Installing unicode-display_width 1.3.0
Fetching sigdump 0.2.4
Installing sigdump 0.2.4
Installing ruby-progressbar 1.8.1
Fetching thread 0.1.7
Fetching sprockets-export 1.0.0
Installing sprockets-export 1.0.0
Fetching sqlite3 1.3.13
Installing thread 0.1.7
Fetching stackprof 0.2.10
Installing sqlite3 1.3.13 with native extensions
Installing stackprof 0.2.10 with native extensions
Fetching turbolinks-source 5.0.3
Fetching websocket 1.2.4
Installing turbolinks-source 5.0.3
Installing websocket 1.2.4
Fetching rainbow 2.2.2
Fetching sucker_punch 2.0.2
Installing sucker_punch 2.0.2
Installing rainbow 2.2.2 with native extensions
Fetching minitest-server 1.0.4
Installing sequel 4.49.0
Installing minitest-server 1.0.4
Fetching tzinfo 1.2.3
Fetching nokogiri 1.8.0
Fetching websocket-driver 0.6.5
Installing websocket-driver 0.6.5 with native extensions
Installing tzinfo 1.2.3
Fetching rack-test 0.7.0
Fetching sprockets 3.7.1
Installing rack-test 0.7.0
Fetching rack-cache 1.7.0
Installing sprockets 3.7.1
Installing rack-cache 1.7.0
Fetching rack-protection 2.0.0
Fetching vegas 0.1.11
Installing vegas 0.1.11
Fetching bunny 2.6.6
Installing rack-protection 2.0.0
Fetching addressable 2.5.1
Installing bunny 2.6.6
Fetching parser 2.4.0.0
Installing addressable 2.5.1
Fetching mime-types 3.1
Installing parser 2.4.0.0
Installing mime-types 3.1
Fetching faraday 0.13.1
Fetching backburner 1.4.1
Installing faraday 0.13.1
Installing backburner 1.4.1
Fetching aws-sdk-core 3.3.0
Fetching uglifier 3.2.0
Installing aws-sdk-core 3.3.0
Installing uglifier 3.2.0
Fetching coffee-script 2.4.1
Fetching em-socksify 0.3.1
Installing em-socksify 0.3.1
Fetching thin 1.7.2
Installing thin 1.7.2 with native extensions
Fetching bootsnap 1.1.2
Installing bootsnap 1.1.2 with native extensions
Fetching childprocess 0.7.1
Installing childprocess 0.7.1
Using rb-inotify 0.9.9 from https://github.com/matthewd/rb-inotify.git (at close-handling@856730a)
Fetching em-hiredis 0.3.1
Installing em-hiredis 0.3.1
Fetching logging 2.2.2
Installing logging 2.2.2
Fetching representable 3.0.4
Installing representable 3.0.4
Fetching mocha 0.14.0
Installing mocha 0.14.0
Using queue_classic 3.2.0.RC1 from https://github.com/QueueClassic/queue_classic.git (at master@cde82d1)
Fetching redis-namespace 1.5.3
Installing redis-namespace 1.5.3
Using sdoc 1.0.0.rc2 from https://github.com/robin850/sdoc.git (at upgrade@0e34035)
Fetching serverengine 1.5.11
Installing serverengine 1.5.11
Fetching turbolinks 5.0.1
Installing coffee-script 2.4.1
Installing turbolinks 5.0.1
Using websocket-client-simple 0.3.0 from https://github.com/matthewd/websocket-client-simple.git (at close-race@e161305)
Fetching minitest-bisect 1.4.0
Using activesupport 5.2.0.alpha from source at `.`
Fetching et-orbi 1.0.5
Installing et-orbi 1.0.5
Fetching faye-websocket 0.10.7
Installing minitest-bisect 1.4.0
Fetching sinatra 2.0.0
Installing faye-websocket 0.10.7
Fetching sidekiq 5.0.4
Installing sinatra 2.0.0
Fetching mail 2.6.6
Installing sidekiq 5.0.4
Fetching rubocop 0.49.1
Installing mail 2.6.6
Fetching faraday_middleware 0.12.2
Installing faraday_middleware 0.12.2
Fetching signet 0.7.3
Installing signet 0.7.3
Fetching google-cloud-env 1.0.1
Installing google-cloud-env 1.0.1
Fetching aws-sdk-kms 1.1.0
Installing aws-sdk-kms 1.1.0
Fetching em-http-request 1.1.5
Installing rubocop 0.49.1
Fetching selenium-webdriver 3.5.1
Installing em-http-request 1.1.5
Fetching listen 3.1.5
Installing listen 3.1.5
Fetching sass-listen 4.0.0
Installing sass-listen 4.0.0
Fetching sneakers 2.5.0
Installing sneakers 2.5.0
Fetching globalid 0.4.0
Installing globalid 0.4.0
Using activemodel 5.2.0.alpha from source at `.`
Fetching delayed_job 4.1.3
Installing delayed_job 4.1.3
Fetching rufus-scheduler 3.4.2
Installing selenium-webdriver 3.5.1
Fetching resque 1.27.4
Installing rufus-scheduler 3.4.2
Fetching googleauth 0.5.3
Installing googleauth 0.5.3
Fetching aws-sdk-s3 1.2.0
Installing aws-sdk-s3 1.2.0
Fetching faye 1.2.4
Installing faye 1.2.4
Fetching sass 3.5.1
Installing resque 1.27.4
Using activejob 5.2.0.alpha from source at `.`
Using activerecord 5.2.0.alpha from source at `.`
Fetching blade-sauce_labs_plugin 0.7.2
Installing sass 3.5.1
Fetching google-api-client 0.13.1
Installing google-api-client 0.13.1
Fetching google-cloud-core 1.0.0
Installing google-cloud-core 1.0.0
Fetching blade 0.7.1
Installing blade 0.7.1
Fetching resque-scheduler 4.3.0
Installing resque-scheduler 4.3.0
Fetching delayed_job_active_record 4.1.2
Installing delayed_job_active_record 4.1.2
Fetching google-cloud-storage 1.4.0
Installing google-cloud-storage 1.4.0
Installing nokogiri 1.8.0 with native extensions
Installing blade-sauce_labs_plugin 0.7.2
Fetching rails-dom-testing 2.0.3
Fetching loofah 2.0.3
Fetching azure-core 0.1.11
Installing loofah 2.0.3
Fetching xpath 2.1.0
Installing azure-core 0.1.11
Installing rails-dom-testing 2.0.3
Fetching kindlerb 1.2.0
Fetching w3c_validators 1.3.3
Installing xpath 2.1.0
Fetching rails-html-sanitizer 1.0.3
Installing kindlerb 1.2.0
Fetching azure-storage 0.12.3.preview
Installing rails-html-sanitizer 1.0.3
Fetching capybara 2.15.1
Installing w3c_validators 1.3.3
Using actionview 5.2.0.alpha from source at `.`
Using actionpack 5.2.0.alpha from source at `.`
Using actioncable 5.2.0.alpha from source at `.`
Using actionmailer 5.2.0.alpha from source at `.`
Using activestorage 5.2.0.alpha from source at `.`
Using railties 5.2.0.alpha from source at `.`
Fetching sprockets-rails 3.2.0
Installing capybara 2.15.1
Installing azure-storage 0.12.3.preview
Installing sprockets-rails 3.2.0
Fetching coffee-rails 4.2.2
Fetching jquery-rails 4.3.1
Using rails 5.2.0.alpha from source at `.`
Using sass-rails 5.0.6 from https://github.com/rails/sass-rails.git (at 5-0-stable@bb5c1d3)
Installing coffee-rails 4.2.2
Installing jquery-rails 4.3.1
Bundle complete! 62 Gemfile dependencies, 178 gems now installed.
Bundled gems are installed into ./vendor/bundle.
Post-install message from sucker_punch:
Sucker Punch v2.0 introduces backwards-incompatible changes.
Please see https://github.com/brandonhilkert/sucker_punch/blob/master/CHANGES.md#20 for details.
+ docker-compose exec -T postgres psql -c 'create database activerecord_unittest;' -U postgres
CREATE DATABASE
+ docker-compose exec -T postgres psql -c 'create database activerecord_unittest2;' -U postgres
CREATE DATABASE
+ set +x
Setup finished!
Now you can attach the ruby container via `docker-compose exec ruby bash`
Enjoy development rails core!
myretina:rails yahonda$ docker-compose exec ruby bash
app@585916b322de:/app$ id
uid=1000(app) gid=1000(app) groups=1000(app)
app@585916b322de:/app$ ruby -v
ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-linux]
app@585916b322de:/app$
```
</details>

Thank you for your help. Closing it.
